src/pydata_sphinx_theme/__init__.py

```diff
@@ -196,10 +196,11 @@ def update_config(app):
     ]
     # Add extra icon links entries if there were shortcuts present
     # TODO: Deprecate this at some point in the future?
+    icon_links = theme_options.get("icon_links", [])
```

Comment:
This LGTM. Just wondering if making an empty list be the default in `theme.conf` would also have worked / is there an advantage to handling it this way?

Reply:
I wanna say it's not possible to make a default an empty list? I think our only options there are strings and none...i could be wrong though

Reply:
I remember trying and ends up with the same conclusion